### PR TITLE
Add examples

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+`bower install next-type --save`
+
+### Import into your Sass
+
+`@import "next-type/main";`
+
+### Sass mixins
+
+Text sans:
+`@include nTypeTextSans([1-5])`
```
Comment (commuterjoy) [inline README.md]: Maybe an example of how to set the colour too?

Comment (keirog) [inline README.md]: Not sure I understand why… This is the type module, so may be better having that as part of the colour module, no?

Comment (commuterjoy) [inline README.md]: I just mean, what SASS would I need to write if I wanted a pink headline?

Comment (commuterjoy) [inline README.md]: Eg, 

```
.headline {
    @include nTypeTextSans(3)
    color: #1e1e1e;
}
```

**Pull request merged**